lib/elixir/pages/Compatibility and Deprecations.md

```diff
@@ -1,6 +1,46 @@
-# Deprecations
+# Compatibility and Deprecations
 
-## Policy
+## Compatibility between Elixir and Erlang/OTP
+
+Erlang/OTP versioning is independent from the versioning of Elixir. Each version of Elixir supports a specific range of Erlang/OTP versions. The compatibility table is shown below.
+
+### Table of Erlang/OTP compatibility
+
+Elixir version | Supported Erlang/OTP versions
+:------------- | :----------------------------
+v1.0.0         | 17
+v1.0.1         | 17
+v1.0.2         | 17
+v1.0.3         | 17
+v1.0.4         | 17
+v1.0.5         | 17-18
+v1.1.0         | 17-18
+v1.1.1         | 17-18
+v1.2.0         | 18
+v1.2.1         | 18
+v1.2.2         | 18
+v1.2.3         | 18
+v1.2.4         | 18
+v1.2.5         | 18
+v1.2.6         | 18-19
+v1.3.0         | 18-19
+v1.3.1         | 18-19
+v1.3.2         | 18-19
+v1.3.3         | 18-19
+v1.3.4         | 18-19
+v1.4.0         | 18-19
+v1.4.1         | 18-19
+v1.4.2         | 18-19
+v1.4.3         | 18-19
+v1.4.4         | 18-19
+v1.4.5         | 18-20
+v1.5.0         | 18-20
+v1.5.1         | 18-20
+v1.5.2         | 18-20
```

Comment:
I mean here is like when you're writing resume, for current employment, you would not write as `beginyear - 2017`, you would rather write as `beginyear - present`  because the `- 2017` would convey wrong information (ended employment).

since you would have to fix that at least once later, why don't you leave it open now; till later when a max supported version for v1.5.2 is determined?

Reply:
This the list of OTP versions that we support. Supported means that it's guaranteed to work or it's a bug. We can't support a version until it's released so we are not listing 21 or leaving it open.

If you look through the changelogs you will see that support is explicitly added for new OTP versions and that usually changes are required until we can support a new version. None of the Elixir versions we list here have been upgraded in their OTP support afaik.